sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/implementation/models/RandomBytes.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.security.keyvault.keys.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.util.Base64Url;
+import com.azure.core.util.CoreUtils;
+import com.azure.security.keyvault.keys.KeyAsyncClient;
+import com.azure.security.keyvault.keys.KeyClient;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A class containing random bytes obtained by calling {@link KeyClient#getRandomBytes(int)} or
+ * {@link KeyAsyncClient#getRandomBytes(int)}.
+ */
+@Fluent
+public class RandomBytes {
+    /*
+     * The bytes encoded as a base64url string.
+     */
+    @JsonProperty(value = "value")
+    private Base64Url bytes;
+
+    /**
+     * Get the bytes encoded as a base64url string.
```

Comment:
Is that true? Just making sure - I don't know the Java codebase well but it looks like we're decoding and returning the raw bytes right and just storing it internally as base64url string right?

Reply:
I got this class from using AutoRest, but we can definitely make it a byte array, as long as we clone the contents to make it effectively immutable.

Reply:
Oh sorry - I just want to clarify what I meant: the doc-comment says it returns a base64url string but the signature says it returns a byte[], so I thought the doc-comment was inaccurate

Reply:
Oh thanks for that. I will update the doc comment to make sure it matches what's happening.